graphenebase/ecdsa.py

```diff
@@ -92,13 +142,9 @@ def sign_message(message, wif, hashfn=hashlib.sha256):
         message = bytes(message, "utf-8")
 
     digest = hashfn(message).digest()
-
-    if sys.version > '3':
-        p = bytes(PrivateKey(wif))
-    else:
-        p = bytes(PrivateKey(wif).__bytes__())
-
-    if USE_SECP256K1:
+    priv_key = PrivateKey(wif)
+    if SECP256K1_MODULE == "secp256k1":
+        p = bytes(priv_key)
```

Comment:
This may break python2 compatibility. That's the reason for the block using `__bytes__()` a few lines above